lib/model/yake_keywords.py

```diff
@@ -53,15 +56,28 @@ def run_yake(self, text: str,
         :param num_of_keywords: int
         :returns: str
         """
+        # TODO: loop over passed in items, or actually trigger processing in batch?
```

Comment:
This looks fine as a first pass on the idea imo

Reply:
@ahmednasserswe adding you here for visibility, let us know if you have any thoughts about this file specifically or the refactor as a whole.